molecule/scenario_resources/tests/test_default.rb

```diff
@@ -8,3 +8,28 @@
     it { should be_enabled }
   end
 end
+
+'pulpcore-webserver' do |webserver|
+  describe port(80) do
+    it { should be_listening }
+  end
+
+  describe port(443) do
+    it { should be_listening }
+  end
+
+  describe http('http://localhost/pulp/api/v3/status',
+                 ssl_verify: false, max_redirects: 1) do
```

Comment:
If we set `max_redirect: 0` here, would we be able to assert that it redirects?

Reply:
@daviddavis  I can create a third test, with `max_redirects: 0` and check if the status eq 301 (without checking body content). Is that what you meant ?

Reply:
Me not being David, that was what i mean. We could assert on the address in the redirect.
I would maybe replace this one, because we test the redirected to address explicitly below.

Reply:
ahah sorry wrong person being called out. Yes it was meant for you.